list_of_subprojects.md

```diff
@@ -0,0 +1,71 @@
+
+- Services
+  - In general Jupyter services such as Binder, nbviewer, and the Jupyter website involve legal, financial, and operational matters. As such, we are proposing that the actual services are managed by working groups that report to the Board of Directors, who can provide the needed support and oversight. For example, if we want to hire full or part time staff to maintain or operate these services, the Board of Directors would be responsible for raising funds, hiring, and managing those staff.
+  - The JupyterHub and Binder teams have a number of unique considerations. Currently all Binder repositories are under the Jupyterhub organizations, but there is a separate (but highly overlapping team) listed as the Binder team. These teams will need to work out if they would like to each have a formal decision making body with an SSC representative, or if they would like to have a single team. The Governance Working Group will be available to the JupyterHub and Binder teams to help working through these questions.
+  - We propose that the Jupyer website and its repository will be governed by a working group that reports to the Board of Directors.
+  - We propose that the nbviewer service (only the actual live service) be governed by a working group that reports to the Board of Directors. The reusable part of nbviewer contained (https://github.com/jupyter/nbviewer) will be governed by the Jupyter Enterprise Subproject.
+- Jupyter Kernels
+  - To consolidate the project’s work on first-party kernels, we propose to establish a new official Subproject called _Jupyter Kernels_ and create a Github organization named jupyter-kernels for the work of the Subproject. All Xeus repositories (https://github.com/jupyter-xeus) will be moved into this organization. This Subproject will also govern the IPython GitHub organization, which will be left in place (https://github.com/ipython).
+  - A new decision making body for this organization will be established, and they will elect an SSC delegate.
+- Jupyter Standards
+  - Ultimately, because Jupyter standards are cross-project in nature, they are owned by the SSC. The mechanics of day-to-day management of the JEP repo will be decided by the SSC.
+  - There are a number of repositories that encode project-wide standards and protocols. To consolidate work on these repositories, we propose to establish a new official Subproject called Jupyter Standards and create a Github organization named jupyter-standards for the work of the Subproject. The following repositories will be moved into this organization:
+    - Jupyter Client (https://github.com/jupyter/jupyter_client)
+    - Jupyter Notebook Format (https://github.com/jupyter/nbformat)
+    - Documentation for other specifications maintained by other Subprojects, such as the Jupyter Widgets message specification, or the Jupyter Server REST APIs.
+    - JEPs repo (https://github.com/jupyter/enhancement-proposals).
+- Jupyter Enterprise
```

Comment:
I find this to be a kind of weird category and I’m not quite sure what it means. Does it imply that anything that’s not in this category is “not enterprise”? I’d disagree with that statement. Is there a more precise description than just calling it enterprise? Or maybe I am just not understanding the word enterprise well?

Reply:
We're really talking about "Enterprise Software" [in the wikipedia sense](https://en.wikipedia.org/wiki/Enterprise_software): use cases that are organizational rather than individual, and thus justify the extra complexity of say docker/k8/etc, rather than just `conda install jupyterlab`. Since the term is widely used for that scenario we'll keep it, but I'll add a clarifying reference to signal that it has nothing to do with corporate/industrial/for-profit usage. Let us know if that clarifies sufficiently.